### ISSUE TITLE
Need help for reproducing linear eval results with published weights.

### ISSUE THREAD

**Issue by lucasb-eyer** [facebookresearch/dino#149]
Hi Mathilde,

I'm porting your models to JAX for further research. I have replicated it so that I get **exactly** the same outputs (logits, and all intermediates too) when passing the same random input to PyTorch and JAX on the same machine, loading the weights you published for ViT-B/16+linear. (I had to get some details right, like GELU being different between the frameworks.) However, when running full evaluation on ImageNet val using the JAX model, I'm getting `77.73%` (resize=bilinear) or `77.548%` (resize=bicubic), when I think I should get `78.162` according to [these eval logs](https://dl.fbaipublicfiles.com/dino/dino_vitbase16_pretrain/dino_vitbase16_pretrain_eval_linear_log.txt).

Thus, AFAICT, the only possible remaining causes for difference would be (1) before the model, i.e. preprocessing of images, or (2) after the model, i.e. computation of score, or (3) something not matching on your side.

For (1), I read [the code](https://github.com/facebookresearch/dino/blob/d2f3156bb34c32ae54312de7fe4e31580f12ff7f/eval_linear.py#L65-L83) and believe I'm doing the same, including setting resize mode to bicubic and antialias=False. However, it is TF resize, so that may make a difference. 
![image](https://user-images.githubusercontent.com/1476029/138895539-a4aceb24-6e55-4f9a-af84-dedcab9cf35d.png)

(2) is trivial, I'm sure I got it right.

(3) I don't have ImageNet for PyTorch at hand and it would take me a while to get it.

Could you please double-check the number with your published weights and code for me? And do you have any more ideas, or would you ascribe it to different framework's resize functions at this point?
